docs/faq.rst

```diff
@@ -78,9 +78,7 @@ KSQL currently supports formats:
 
 -  DELIMITED (e.g. comma-separated value)
 -  JSON
--  Avro (requires Confluent Schema Registry and setting ``ksql.schema.registry.url`` in the KSQL configuration file)
-
-*Support for Apache Avro is expected soon.*
+-  Avro (requires Confluent Schema Registry and :ref:`setting ``ksql.schema.registry.url`` in the KSQL configuration file <install_ksql-avro-schema>`)
```

Comment:
Nit: in the KSQL *server* configuration file

I want us to be extra clear on this because it might still be confusing what you can/should configure in the server vs. CLI configs.